build/updatehistorictzdb.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -e
+
+rm -rf tmp
+mkdir tmp
+
+# Get the list of files, just once.
+
+mkdir -p tmp/releases
+cd tmp/releases
+FILES=files.txt
+
+gsutil.cmd ls gs://nodatime/releases > $FILES
```

Comment:
Could do. At the moment it also relies on nuget and msbuild, neither of which I'm expecting to be callable "just like that" for a little while.